x-pack/plugins/cases/public/components/recent_cases/recent_cases.tsx

```diff
@@ -33,12 +33,14 @@ const MarkdownContainer = styled.div`
 `;
 
 const TruncateComp = styled.div`
-  text-overflow: ellipsis;
-  display: -webkit-box;
-  -webkit-line-clamp: 3;
-  -webkit-box-orient: vertical;
-  overflow: hidden;
-  word-break: break-word;
+  & .euiMarkdownFormat {
```

Comment:
How was this what made a difference? 😄 I thought the problem was some of the properties below not being supported in safari.

Reply:
yeah that was my quick research but later I found out that
`webkit-line-clamp` is not supported in case there is any kind of animation applied to element, which I fixed in this [PR](https://github.com/elastic/kibana/pull/148561).

However, when there is no animation if these properties are applied to element directly or to immediate parent of a text element then it works in Safari. 😄 
In this case style was applied to `Parent of Parent` which I fixed by applying it to Parent of text element.

Reply:
> However, when there is no animation if these properties are applied to element directly or to immediate parent of a text element then it works in Safari. 😄

Ok, after you mentioned it yesterday I went to check with safari other places in Cases where we use `-webkit-line-clamp` and was surprised that actually they were working fine.

That explains it!